### ISSUE TITLE
Missing data and other questions

### ISSUE THREAD

**Issue by lucaputelli** [fdfdsf/Bio-Extraction#1]
Hello,
I found your paper very interesting and, while I was trying to re-implement your model for another dataset, a few difficulties came up.
First of all, it seems that the ADE corpus is missing. I have found it online (https://github.com/trunghlt/AdverseDrugReaction/tree/master/ADE-Corpus-V2) but in your config file you mention three specific files ("ade_data.json", "train.txt" and "dev.txt") that I can't find. Can you put them online too or tell me where I can find them?
Second, the Entity-Aware Decoding Layer checks which tokens are predicted as entities and sum the corresponding LSTM outputs (h_wi in the paper) into a single vector, which is concatenated do the label embeddings treated in the same way. In case there are _n_ entities in the first sequence, does the Entity-Aware Decoding Layer produce _n_ vectors or a single one just making the sum of all the h_wi?
Last question: are the embedding labels calculated during the training phase or are they pre-trained?

Thank you,
Luca

**Comment by vhientran** [comment]
Hi @fdfdsf , if possible, please provide the ADE corpus. Thank you very much!

**Comment by Prophet37** [comment]
Hi! Could you please provide with the corpus used in this project? Would be very helpful, thank you very much!

**Comment by Prophet37** [comment]
@angelotran05 Were you able to retrieve the dataset?

**Comment by vhientran** [comment]
@Prophet37 , I haven't got it yet...
